File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        dropCount = 0
        n = len(nums)
        for i in range(n):
            if nums[i] > nums[(i+1)%n]:
                dropCount+=1
        if dropCount > 1:
            return False 
        return True 

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var check = function(nums) {
    let dropCount = 0;
    let n = nums.length;
    for(let i = 0;i<nums.length;i++) {
        if(nums[i] > nums[(i+1)%n]) {
            dropCount +=1;
        }
    }
    if(dropCount > 1) return false 
    return true 
};